staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -178,6 +178,7 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		endpointsLister: s.endpointsLister,
 	}
 	s.GenericAPIServer.HandlerContainer.Handle("/apis", apisHandler)
+	s.GenericAPIServer.HandlerContainer.Handle("/apis/", apisHandler)
```

Comment:
> This provides consistency with line 224, I think.

That line does something different.  It's more narrowly scoped.



> this has the side-effect of making any subpath of /apis/ (like /apis/foo/bar/baz) be handled by apisHandler. Is that what we want?

There's a double check to make sure that we delegate through if there's a subpath.